src/modules/components/vote/voteDataListItem/voteDataListItemStructure/voteDataListItemStructure.tsx

```diff
@@ -58,13 +58,13 @@ export const VoteDataListItemStructure: React.FC<IVoteDataListItemStructureProps
             <MemberAvatar address={voter.address} ensName={voter.name} responsiveSize={{ md: 'md' }} />
             <div className={centerInfoClassNames}>
                 <span className="flex items-center gap-x-1 text-neutral-800 md:gap-x-1.5">
-                    {resolvedUserHandle}
+                    <span className="truncate">{resolvedUserHandle}</span>
```

Comment:
Could you add a story here as well for long user address/ens and/or symbol? I liked the ones for the other components as we can easily double check if truncation still works 👍🏻